src/controllers/VariableController.ts

```diff
@@ -100,6 +100,20 @@ export class VariableController {
         return res.setVariableType(variableId, type).then((result) => getEditorResponseData<null>(result));
     };
 
+    /**
+     * This method sets the items of the list variable
+     * 
+     * The items need to be unique and are not case sensitive. 
+     *
+     * @param variableId the id of the list variable
+     * @param items the items of the list
+     * @returns
+     */
+    setListVariableItems = async (variableId: string, items: string[]) => {
```

Comment:
Maybe we could rename it to "setListVariable" to be in the same style as "setImageVariable" or "set<TYPE>Variable"? Appending "items" seems redundant as ListVariable is a list of elements (in this case items). Just a thought taking into the account the SDK rewrite in my pr.